pkg/ignition/config.go

```diff
@@ -67,6 +76,31 @@ func File(config *Config) (string, error) {
 			return "", err
 		}
 	}
+	if len(config.DnsServers) > 0 {
```

Comment:
No need to check for 0 here as you are having the for loop below.

Reply:
in for loop i am just combining all the IPs, if i remove this check default content will get added to ignition content without IPs like this 
 - path: /etc/systemd/resolved.conf.d/dns.conf
      mode: 0644
      contents:
        inline: |
          [Resolve]
 Or i can move this check before forming the dnsConf content at line no. 86